plugins/aws/cloudtrail/cloudtrailBucketDelete.js

```diff
@@ -45,6 +55,13 @@ module.exports = {
                 // Skip CloudSploit-managed events bucket
                 if (trail.S3BucketName == helpers.CLOUDSPLOIT_EVENTS_BUCKET) return cb();
 
+                if (!listBuckets.data.find(bucket => bucket.Name == trail.S3BucketName)) {
+                    helpers.addResult(results, 2,
+                        'Unable to locate S3 bucket, it may has been deleted',
```

Comment:
```suggestion
                        'Unable to locate S3 bucket, it may have been deleted',
```
```suggestion
                        'Unable to locate S3 bucket, it may has been deleted',
```